### PR TITLE
Refactor EVP-LRQA-82925 | Master

### PR DESCRIPTION
**Pull request by MGabriellaS**
[LRQA-82925](https://liferay.atlassian.net/browse/LRQA-82925)

### CODE REVIEW

**Comment by MGabriellaS** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganizationStatus.testcase [hunk 1]
```diff
@@ -26,29 +26,26 @@ definition {
 		}
 
 		task ("Employee user creates an organization") {
```
Comment (heiditrindade) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganizationStatus.testcase:28]: Change to "Create an organization"

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganizationStatus.testcase [hunk 2]
```diff
@@ -117,10 +76,10 @@ definition {
 		}
 	}
 
-	@description = "LPS-164733 - Verify that Only the Finance user can not see 'Review button' when organization are in 'Awaiting Approval On EVP' status"
+	@description = "LPS-164733 - Verify that the Finance user can not see 'Review button' when organization are in 'Awaiting Approval On EVP' status"
```
Comment (heiditrindade) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganizationStatus.testcase:79]: Fix to "LPS-164733 - Verify that the Finance user can not see 'Review button' when organization is in 'Awaiting Approval On EVP' status"

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganizationStatus.testcase (hunk 1 above)
Comment (MGabriellaS) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganizationStatus.testcase:28]: I already update this on this [PR](https://github.com/is-solutions-delivery/liferay-portal/pull/4985)

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganizationStatus.testcase (hunk 2 above)
Comment (MGabriellaS) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganizationStatus.testcase:79]: I already update this on this https://github.com/is-solutions-delivery/liferay-portal/pull/4985

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganizationStatus.testcase [hunk 3]
```diff
@@ -65,48 +62,10 @@ definition {
...
-
-			MenuItem.click(menuItem = "Details");
-		}
-
-		task ("Then the manager user can see the comment 'Your organization has been approved!' in the details page") {
-			EVPUtils.assertHeadersTitle(listHeaderTitle = "Organization Test,Your organization has been approved!");
-		}
-	}
-
 	@description = "LPS-164733 - Verify that Only the Employee user can not see 'Review button' when organization are in 'Awaiting Approval On EVP' status"
```
[the remark this answers is not in the document]
Comment (MGabriellaS) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganizationStatus.testcase:65]: I already update this on this https://github.com/is-solutions-delivery/liferay-portal/pull/4985

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganizationStatus.testcase (hunk 2 above)
Comment (MGabriellaS) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganizationStatus.testcase:79]: I already update this on this https://github.com/is-solutions-delivery/liferay-portal/pull/4985

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganizationStatus.testcase [hunk 4]
```diff
@@ -117,10 +76,10 @@ definition {
 		}
 	}
 
-	@description = "LPS-164733 - Verify that Only the Finance user can not see 'Review button' when organization are in 'Awaiting Approval On EVP' status"
+	@description = "LPS-164733 - Verify that the Finance user can not see 'Review button' when organization are in 'Awaiting Approval On EVP' status"
 	@priority = 5
 	test FinanceCanNotSeeReviewInAwaitingEVPStatus {
-		task ("Given that login with Finance user") {
+		task ("Given that Finance user is logged") {
```
[the remark this answers is not in the document]
Comment (MGabriellaS) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganizationStatus.testcase:82]: I already update this on this https://github.com/is-solutions-delivery/liferay-portal/pull/4985

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganizationStatus.testcase [hunk 5]
```diff
@@ -139,74 +98,54 @@ definition {
...
-
-			EVPNavigator.goToNavItem(navItem = "Organization");
+		task ("Given that the created Organization has already been approved by Manager user") {
+			EVPUtils.managerApproval(
+				listingPage = "Organization",
+				organizationName = "Organization Test");
 		}
 
-		task ("When the manager approve the organization created") {
+		task ("When the manager user clicks on kebab menu of a Organization with 'Awaiting Finance Approval' status") {
```
[the remark this answers is not in the document]
Comment (MGabriellaS) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganizationStatus.testcase:136]: I already update this on this https://github.com/is-solutions-delivery/liferay-portal/pull/4985

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganizationStatus.testcase [hunk 6]
```diff
@@ -139,74 +98,54 @@ definition {
...
+		task ("Then the manager user can't see the Review option") {
 			MenuItem.viewNotPresent(menuItem = "Review");
 		}
 	}
 
 	@description = "LPS-164733 - Verify that the EVP Staff can Reject a new organization"
 	@priority = 5
 	test ManagerCanRejectOrganization {
-		task ("Given that Sign as Manager user") {
+		task ("Given that Manager user is logged") {
```
[the remark this answers is not in the document]
Comment (MGabriellaS) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganizationStatus.testcase:148]: I already update this on this https://github.com/is-solutions-delivery/liferay-portal/pull/4985

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPHomePage.testcase [hunk 7]
```diff
@@ -38,29 +38,31 @@ definition {
...
 			User.logoutPG();
-		}
 
-		task ("Then the employee should not be able to view the Request tab") {
 			var portalURL = PropsUtil.get("portal.url");
 
 			Navigator.openSpecificURL(url = "${portalURL}/web/evp-portal");
+		}
 
+		task ("Then the user can't the Request tab") {
```
[the remark this answers is not in the document]
Comment (MGabriellaS) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPHomePage.testcase:65]: thanks for the review. I will update this

**Comment by MGabriellaS** [comment]
ci:test:sf

**Review by vinicius-paes — APPROVED** [review]

**Comment by vinicius-paes** [comment]
ci:forward

**Pull request closed without merging**